docs/handbook/04_streams.md

```diff
@@ -66,7 +66,7 @@ def destroy
   @message.destroy
 
   respond_to do |format|
-    format.turbo_stream { render turbo_stream: turbo_stream.remove dom_id(@message) }
+    format.turbo_stream { render turbo_stream: turbo_stream.remove(ActionView::RecordIdentifier.dom_id @message) }
```

Comment:
Reading through the [`Turbo::Streams::TagBuilder#extract_target_name_from`](https://github.com/hotwired/turbo-rails/blob/6df29f72d6e05b02f83a1310bad9b87c051c8e94/app/models/turbo/streams/tag_builder.rb#L113-L119) invoked by `turbo_stream.remove`, I think omitting the `dom_id` entirely might be an improvement:

```suggestion
    format.turbo_stream { render turbo_stream: turbo_stream.remove @message }
```

Reply:
thanks!

`render turbo_stream: turbo_stream.remove @message` is raise error,
so I will change `render turbo_stream: turbo_stream.remove(@message)` here!